TESTS/store_test.js

```diff
@@ -1,18 +1,18 @@
-let registratedUser = {
+let registerUser = {
     firstName: 'Ann',
     lastName: 9,
     email: 'qatest1287@gmail.com',
     telephone: 4372552055,
     password: 'Testing987654!',
-}
+};
 Feature('store');
 
-Scenario ('test eng',  ({ I, homePagePage, registerPage }) => {
-    I.openStore();
-    homePagePage.selectWebsiteEngLang();
-    homePagePage.openRegistrationPage();
+Scenario('register new user',  ({ I, homePagePage, registerPage }) => {
+    I.openStore(); 
+    I.selectWebsiteEngLang();  //homePagePage.
+    homePagePage.openRegistrationPage();//
     registerPage.verifyRegisterAccountText();
-    registratedUser.email = Date.now() + '@test.com';
-    registerPage.fillRegistrationDetails(registratedUser);
-    pause();
+    registerUser.email = Date.now() + '@test.com';
+    registerPage.fillRegistrationDetails(registerUser);
```

Comment:
`fillRegistrationDetails()` function not only fills details, but completes registration. name it `registerUser()`